lib/fluent/plugin/out_google_cloud.rb

```diff
@@ -193,6 +200,8 @@ def configure(conf)
       @common_labels = {}
       @common_labels.merge!(@labels) if @labels
 
+      @invalid_tag_regexp = %r{[^[:alpha:]\d/._-]}
```

Comment:
This is the only accepted characters by the Logging API right now. If we let users customize this tag, doesn't that mean the converted tags could still get rejected by the Logging API?